### PR TITLE
Add new records to yunexiz.is-a.dev (Im not 100% sure if i did this correctly)

### PR DESCRIPTION
**Pull request by Yunexiz**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/yunexiz.json [hunk 1]
```diff
@@ -5,5 +5,7 @@
     },
     "record": {
         "A": ["109.71.252.37"]
+        "TXT": ["v=spf1 include:spf.improvmx.com ~all"]
```
Comment (VaibhavSys) [inline domains/yunexiz.json:8]:
```suggestion
        "TXT": ["v=spf1 include:spf.improvmx.com ~all"],
```

File: domains/yunexiz.json [hunk 2]
```diff
@@ -5,5 +5,7 @@
     },
     "record": {
         "A": ["109.71.252.37"]
```
Comment (VaibhavSys) [inline domains/yunexiz.json:7]:
```suggestion
        "A": ["109.71.252.37"],
```

File: domains/yunexiz.json [hunk 3]
```diff
@@ -5,5 +5,7 @@
...
+        "TXT": ["v=spf1 include:spf.improvmx.com ~all"]
+        "MX": ["mx1.improvmx.com.", "mx2.improvmx.com."]
```
Comment (VaibhavSys) [inline domains/yunexiz.json:9]:
```suggestion
        "MX": ["mx1.improvmx.com", "mx2.improvmx.com"]
```

**Review by VaibhavSys — DISMISSED** [review]
Invalid Syntax and Error in MX

**Review by wdhdev — APPROVED** [review]

**Pull request merged**